lib/elixir/lib/calendar/iso.ex

```diff
@@ -343,9 +344,8 @@ defmodule Calendar.ISO do
   def from_unix(integer, unit) when is_integer(integer) do
     total = System.convert_time_unit(integer, unit, :microsecond)
     if total in @unix_range_microseconds do
-      microsecond = rem(total, 1_000_000)
```

Comment:
I wonder if a better fix would be to replace `rem` here by `Integer.mod` and `div` below by `Integer.floor_div`. I don't think we need to change `iso_microseconds_to_datetime`. :)

Reply:
Ah yes! That's much better. :+1: